index.js

```diff
@@ -3,63 +3,55 @@ module.exports = {
   extends: "eslint:recommended",
   plugins: ["flowtype", "babel"],
```

Comment:
Is the babel plugin included in the preset just in case a consumer extending off this preset wants to use the rules it provides?